omnigibson/utils/motion_planning_utils.py

```diff
@@ -118,12 +123,17 @@ def create_state(space, x, y, yaw):
         state().setYaw(_wrap_angle(yaw))
         return state
 
-    def state_valid_fn(q):
+    def state_valid_fn(q, verbose=True):
+        """
+        returns if the input pose is in collision with any objects within the context
+        verbose (bool): Whether the action primitive is testing the sampling point (default to true), or the pose is used for navigation (false)
```

Comment:
verbose should be defined as whether the collision detector should output information about collisions or not. you can explain that in the sampling mode verbose is too noisy so this defaults to False. Note that the default value, and the values in the explanation, are both wrong.